src/gradualizer_type.erl

```diff
@@ -308,6 +313,8 @@
 -type gr_type_var() :: atom() | string().
 -type af_type_variable() :: {'var', anno(), gr_type_var()}.
 
+-type gr_rigid_type_variable() :: {'rigid_var', anno(), atom()}.
```

Comment:
Since most of this file is copied from OTP, we want to explain all our changes with comments. The prefix "gr" is good but maybe something more about why we want to add this type?

Reply:
Good point, I added the missing comment, together with a forgotten doc for the `make_rigid_type_vars/1` (with a slightly different wording): 5a18b8eb. Is it better now?